docs/README.md

```diff
@@ -16,7 +16,250 @@ The Sail-operator does not manage Gateways. You can deploy a gateway manually ei
+          --from-file=certs/west/root-cert.pem \
+          --from-file=certs/west/cert-chain.pem
+    ```
+
+4. Create Sail CR on east
+
+    ```sh
+    kubectl apply --context "${CTX_CLUSTER1}" -f - <<EOF
+    apiVersion: operator.istio.io/v1alpha1
+    kind: Istio
+    metadata:
+      name: default
+    spec:
+      version: v1.22.0
+      namespace: istio-system
+      values:
+        pilot:
+          resources:
+            requests:
+              cpu: 100m
+              memory: 1024Mi
+        global:
+          meshID: mesh1
+          multiCluster:
+            clusterName: east
+          network: network1
+    EOF
+    kubectl wait --context "${CTX_CLUSTER1}" --for=jsonpath='{.status.revisions.ready}'=1 istios/default --timeout=3m
+    ```
+
+5. Create east-west gateway on east
+
+    ```sh
+    "${ISTIOCTL_DIR}"/samples/multicluster/gen-eastwest-gateway.sh \
+        --mesh mesh1 --cluster east --network network1 | \
+        "${ISTIOCTL_DIR}/bin/istioctl" --context="${CTX_CLUSTER1}" install -y -f -
+    ```
+
+6. Expose services on east
+
+    ```sh
+    kubectl --context "${CTX_CLUSTER1}" apply -n istio-system -f \
+        "${ISTIOCTL_DIR}"/samples/multicluster/expose-services.yaml
+    ```
+
+7. Create Sail CR on west
+
+    ```sh
+    kubectl apply --context "${CTX_CLUSTER2}" -f - <<EOF
+    apiVersion: operator.istio.io/v1alpha1
+    kind: Istio
+    metadata:
+      name: default
+    spec:
+      version: v1.22.0
+      namespace: istio-system
+      values:
+        pilot:
+          resources:
+            requests:
+              cpu: 100m
+              memory: 1024Mi
+        global:
+          meshID: mesh1
+          multiCluster:
+            clusterName: west
+          network: network2
+    EOF
+    kubectl wait --context "${CTX_CLUSTER2}" --for=jsonpath='{.status.revisions.ready}'=1 istios/default --timeout=3m
+    ```
+
+8. Create east-west gateway on west
+
+    ```sh
+    "${ISTIOCTL_DIR}"/samples/multicluster/gen-eastwest-gateway.sh \
+        --mesh mesh1 --cluster west --network network2 | \
+        "${ISTIOCTL_DIR}"/bin/istioctl --context "${CTX_CLUSTER2}" install -y -f -
+    ```
+
+9. Expose services on west
+
+    ```sh
+    kubectl --context "${CTX_CLUSTER2}" apply -n istio-system -f \
+        "${ISTIOCTL_DIR}"/samples/multicluster/expose-services.yaml
+    ```
+
+10. Install a remote secret in west that provides access to east’s API server.
+
+    If using kind, first get the east controlplane ip and pass the `--server` option to `istioctl create-remote-secret`
+
+    ```sh
+    WEST_CONTAINER_IP=$(kubectl get nodes east-control-plane --context "${CTX_CLUSTER1}" -o jsonpath='{.status.addresses[?(@.type == "InternalIP")].address}')
+    "${ISTIOCTL_DIR}"/bin/istioctl create-remote-secret \
+      --context="${CTX_CLUSTER1}" \
+      --name=east \
+      --server="https://${WEST_CONTAINER_IP}:6443" | \
+      kubectl apply -f - --context "${CTX_CLUSTER2}"
```

Comment:
It looks like this is similar to the alternate approach described in Lines 181-184 except that we are passing some additional arguments here. If there is no functional difference between the two, I would suggest that we can remove this and simply document the simpler one.

Reply:
Well this example will only work on kind if you pass the addtional `--server=...` argument. Maybe I can add a note below the example documenting the extra argument for kind instead of showing the same example twice?

Reply:
I switched the examples so that the simpler example shows up first.